faq-v1.md

```diff
@@ -140,10 +142,27 @@ There are other means of ensuring that the device is not used by a separate part
 
 <a name="q-c2"></a>**Q-C2: [SP 800-63C Section 5.2](https://pages.nist.gov/800-63-3/sp800-63c.html#-52-privacy-requirements) states “An IdP MAY disclose information on subscriber activities to other RPs within the federation for security purposes, such as communication of compromised subscriber accounts.” What does this mean?**
 
 **A-C2:** This is our way of saying a provider can share information with federation participants to increase protections for the user and the overall federation network. In a federation, the goal is to share the minimum amount of data about a user to protect their privacy. But SP 800-63C enables an IdP or an RP to disclose information about subscriber activities to other participants within the federation for security purposes, such as communicating the detection of a potentially compromised subscriber account. This approach, typically called a “shared signals model” is becoming more common in the market.
+
 For example, an individual may use an email account to manage their online shopping account. If the email provider detects an anomaly, such as a possible account takeover, they can send this signal to the bank. The bank can do whatever they want with this information based on their risk profile and business rules. In some cases the bank may ignore the signal. In others, or if the bank got similar signals from multiple providers in a short period of time, it might choose to notify the user of suspicious activity associated with their account and use additional security measures the next time the user, or malicious actor, attempts to log in. Sharing information across the federation can help stop hackers from gaining unauthorized access to participating services.
 
+<a name="q-c3"></a>**Q-C3: How do I reach FAL3?
+
+**A-C3:** The highest federation assurance level of FAL3 is intended to be a mostly forward-looking goal for very high value or very high risk situations, and as such takes some effort to get to. In FAL3, the IdP has to provide not only an assertion that's been encrypted to the RP's specific key (as in FAL2), but that assertion also needs to have a reference to a key. This RP has to ask the subscriber to prove possession of that key directly in order to reach FAL3, as this proves that the current user is in fact the one referenced in the assertion itself. 
+
+In practical terms, with today's technology, the easiest way to reach FAL3 is to use a certificate-based authentication (such as PIV) at the IdP and include the certificate reference in the assertion, either as a thumbprint or reference. When the subscriber logs in to the RP, the RP can then challenge the subscriber to present their certificate directly. Importantly, the RP only has to make sure that the certificate is the one in the assertion, and the RP does not have to validate the certificate's chain or trust or process any of the attributes within the certificate. 
```

Comment:
A bit of a nit, but I'd probably refer to it as a key reference rather than a certificate reference (since it's holder-of-key, not certificate). Also would call it a fingerprint, not thumbprint. Might even say "fingerprint (hash of public key)".

Reply:
The specific example here is certificate, but I can reword that to make it clear we're referencing the key from the certificate. And "thumbprint" seems to be the industry term, at least in the OAuth/OIDC world. If there's a reference for the term.

Reply:
OK, "thumbprint" must be an OAuth thing: "key fingerprint" has 557,000 hits on Google and "key thumbprint" has 1,850. And Wikipedia defines it (on the thumbprint disambiguation page) as "a term for public key fingerprint used in Microsoft software," not that they're authoritative or anything. But if that's the correct term to use here, use it, but perhaps it should be defined.

Reply:
I'm fine with "key fingerprint", and I'll change that.

Interestingly, I'm thinking Microsoft must have pushed it into the OAuth/OIDC space -- not surprising, to be honest.

Reply:
Updated in current draft.